Creating a new Reviewing PRs wiki page (#95)
endless-sky/endless-sky-wiki: wiki/ReviewingPRs.md

### Reviewing a pull request

> Reviewing pull requests is an important part of maintaining Endless Sky and the main way that new content is ultimately added to the game. Your feedback helps contributors improve their work and ensures that new content is the best it can be. [This discussion](https://github.com/endless-sky/endless-sky/discussions/10358) covers some great tips for new reviewers.

Reviewer: I don't think the word "ultimately" is necessary here.
```suggestion
Reviewing pull requests is an important part of maintaining Endless Sky and the main way that new content is added to the game. Your feedback helps contributors improve their work and ensures that new content is the best it can be. [This discussion](https://github.com/endless-sky/endless-sky/discussions/10358) covers some great tips for new reviewers.
```

* **Always read the PR description**: Read the whole thing, and you have a better chance of avoiding making mistakes in your review because you didn't have the background.

> * **Don't do reviews as a way to calm down**: Reviewing others work can be calming, but doing reviews when stressed can also lead you to overreact to relatively small changes. Even if you don't, your reviews will probably be worse than usual.

Reviewer: ```suggestion
* **Don't do reviews as a way to calm down**: Reviewing others' work can be calming, but doing reviews when stressed can also lead you to overreact to relatively small changes. Even if you don't, your reviews will probably be worse than usual.
```

* **Use the "request changes" button sparingly, or not at all**: In most cases, leaving comments with suggested changes is enough. Developers typically don’t merge PRs with unresolved comments, so there's usually no need to use "suggest changes."

* **Don't review draft PRs. Draft PRs are still a work in progress**: Any feedback you give is likely to be outdated by the time the PR is ready for review, leading to unwanted notifications and overwritten reviews.

* **If you keep making the same correction, check it**: The internet is great for searching up weird grammar and punctuation rules, especially when it comes to American vs British English. ES' source files are great for examining existing precedents within the game itself.

> * **Review comments for clarity, not conventions**: Comments in data or code files are mainly read by contributors to the repo and are meant to be understood, not grammatically flawless. If you don't understand a comment, by all means suggest a change to it. But if the meaning's clear, there's no need to nitpick small style issues or better yet, remind the PR author in a single summary comment rather than commenting on every instance.

Reviewer 2: ```suggestion
* **Review comments for clarity, not conventions**: Comments in data or code files are mainly read by contributors to the repo and are meant to be understood, not grammatically flawless. If you don't understand a comment, by all means suggest a change to it. But if the meaning's clear, there's no need to nitpick small style issues; better yet, remind the PR author in a single summary comment rather than commenting on every instance.
```

## Reviewing a pull request

Reviewing pull requests is an important part of maintaining Endless Sky and the main way that new content is added to the game. Your feedback helps contributors improve their work and ensures that new content is the best it can be. [This discussion](https://github.com/endless-sky/endless-sky/discussions/10358) covers some great tips for new reviewers.

### Reviewing your first PR

Reviewing PRs is a great first step to getting involved with Endless Sky as a contributor. To begin reviewing your first PR on Endless Sky, please use the resources provided below. 

> 1. Login to your Github account or create one using [this Github guide](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).

Reviewer: I find that Apple autocorrects "Github" to "GitHub". Is there a particular standard to be using?

Reviewer 2: I think 'GitHub' is the official form of capitalization.

Author: Good catch -- 'GitHub' is the official capitalization and the form used elsewhere in the wiki. Just fixed

* **Use the "request changes" button sparingly, or not at all**: In most cases, leaving comments with suggested changes is enough. Developers typically don’t merge PRs with unresolved comments, so there's usually no need to use "suggest changes."

* **Don't review draft PRs. Draft PRs are still a work in progress**: Any feedback you give is likely to be outdated by the time the PR is ready for review, leading to unwanted notifications and overwritten reviews.

* **If you keep making the same correction, check it**: The internet is great for searching up weird grammar and punctuation rules, especially when it comes to American vs British English. ES' source files are great for examining existing precedents within the game itself.

* **Review comments for clarity, not conventions**: Comments in data or code files are mainly read by contributors to the repo and are meant to be understood, not grammatically flawless. If you don't understand a comment, by all means suggest a change to it. But if the meaning's clear, there's no need to nitpick small style issues; better yet, remind the PR author in a single summary comment rather than commenting on every instance.

> * **Partial reviews are still helpful**: You may not be able to review every one of the dozens of files in a large PR. But, you can still look through one or two files and make a couple of suggestions. Pretty much everyone involved will appreciate it, including the final player. 

Reviewer 2: ```suggestion
* **Partial reviews are still helpful**: You may not be able to review every one of the dozens of files in a large PR, but you can still look through one or two files and make a couple of suggestions. Pretty much everyone involved will appreciate it, including the final player. 
```

* **Balance criticism with positive feedback**: Just because you absolutely hate what a PR is trying to accomplish doesn't mean you can't find something good in it. Scathing criticisms are only going to antagonize the author, as well as anyone who actually really liked that PR.

* **Use @ mentions sparingly**: Use it if you're trying to get the attention of people not involved with the PR, if you're writing a PR yourself and want to credit someone else's work, or if someone hasn't responded to your review request or other comments in a while.

> * **You don't have to have an opinion on everything**: Express your opinions, but only when doing so will actual accomplish something productive.

Reviewer 2: ```suggestion
* **You don't have to have an opinion on everything**: Express your opinions, but only when doing so will actually accomplish something productive.
```

## Reviewing a pull request

Reviewing pull requests is an important part of maintaining Endless Sky and the main way that new content is added to the game. Your feedback helps contributors improve their work and ensures that new content is the best it can be. [This discussion](https://github.com/endless-sky/endless-sky/discussions/10358) covers some great tips for new reviewers.

### Reviewing your first PR

Reviewing PRs is a great first step to getting involved with Endless Sky as a contributor. To begin reviewing your first PR on Endless Sky, please use the resources provided below. 

> 1.  Login to your Github account or create one using [this Github guide](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).

Reviewer: ```suggestion
1. Login to your Github account or create one using [this Github guide](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
```

Reviewing PRs is a great first step to getting involved with Endless Sky as a contributor. To begin reviewing your first PR on Endless Sky, please use the resources provided below. 

> 1. Login to your GitHub account or create one using [this GitHub guide](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).

Reviewer 2: ```suggestion
1. Log in to your GitHub account or create one using [this GitHub guide](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
```
